Gems/Atom/Asset/Shader/Config/Platform/Windows/vulkan/shader_build_options.json

```diff
@@ -7,6 +7,7 @@
         ],
         "dxc": ["-spirv"
               , "-fvk-use-dx-layout" // Use the same memory layout as DX12, otherwise some offset of constant may get wrong.
+              , "-fspv-target-env=vulkan1.1"
```

Comment:
We will likely need a similar change for Vulkan Linux also

Reply:
Can we assume we can enforce vulkan 1.1 also for android vulkan?

Reply:
Hmm I honestly don't know the answer to this without a meaningful minspec. Maybe @moudgils has an idea?

Reply:
https://developer.android.com/about/dashboards says 61% of devices have 1.1 so we can contemplate moving to 1.1 but I dont understand how this change affects android. Android config is tracked here (C:\GitHub\o3de\Gems\Atom\Asset\Shader\Config\Platform\Android) so this change should only impact Windows and I agree with Jeremy and update C:\GitHub\o3de\Gems\Atom\Asset\Shader\Config\Platform\Linux\shader_build_options.json as well

Reply:
This sounds like we need an RFC to discuss with the community.

Reply:
Should we provide a way to switch SPD and the existing multiple pass downsampling?

Reply:
I'm not sure I understand the distinction between vk1.1 here and the vulkan 1.1 `spv-target-env`. I think it's possible to enable 1.1 as the SPIR-V target environment while still using extensions to go from vk 1.0 to 1.1 parity but I'm not sure.

> Should we provide a way to switch SPD and the existing multiple pass downsampling?

If it's not too difficult, it would be good I think to allow this and have SPD be the default.

Reply:
Same. Please see my comments for the Linux and Android change. Same applies here. Make the change just for the shader in question